module/zfs/zfs_log.c

```diff
@@ -229,6 +229,10 @@ zfs_log_fuid_domains(zfs_fuid_info_t *fuidp, void *start)
 static int
 zfs_xattr_owner_unlinked(znode_t *zp)
 {
+#ifdef __FreeBSD__
+	/* We don't support this type of XATTR yet so just ignore */
```

Comment:
Are you positive that's the case?  This check is for the directory based xattrs which I thought FreeBSD always supported.  It does not pertain to the system attribute based xattrs added for Linux.